src/views/games/Qwixx.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+	<v-container>
+		<v-row class="mx-4 my-4">
+			<h2 class="display-3 font-weight-black underlined">Qwixx</h2>
+		</v-row>
+		<v-row>
+			<input class="mx-4 my-4" v-model="name" placeholder="Put name here" />
+			<v-btn small color="white">Submit</v-btn>
+		</v-row>
+
+		
+		<v-row class="mb-6">
+			<div class="redBtns mr-3 mb-2" v-for="redBtn in redBtns" :key="redBtn.id">
```

Comment:
I pulled your code and saw errors with key values. I would just change the v-for's to this
v-for="(redBtn, i) in redBtns" :key="i" for all four sections of buttons.